lib/Runtime/Library/ArgumentsObject.h

```diff
@@ -122,7 +127,9 @@ namespace Js
         BOOL IsFormalArgument(PropertyId propertyId);
         BOOL IsFormalArgument(PropertyId propertyId, uint32* pIndex);    // Checks whether property is numeric, and on success sets that index.
         BOOL IsArgumentDeleted(uint32 index) const;
+    public:
         const ActivationObject* const GetFrameObject() { return frameObject; }
+        void SetFrameObject(ActivationObject * value) { frameObject = value; }
```

Comment:
Sure. will try that out.

---
In reply to: [67771245](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67771245) [](ancestors = 67771245)